js/ui/appDisplay.js

```diff
@@ -457,8 +457,8 @@ const AllView = new Lang.Class({
         panAction.connect('gesture-cancel', Lang.bind(this, this._onPanEnd));
         panAction.connect('gesture-end', Lang.bind(this, this._onPanEnd));
         this._panAction = panAction;
-        this._scrollView.add_action(panAction);
         this._panning = false;
+
```

Comment:
I added it on purpose to separate these two blocks, as one is about panning and the other about folders. I'd like to keep it